test/build.js

```diff
@@ -108,6 +108,93 @@ export function buildWithCssModules() {
   })
 }
 
+export function buildWithCssModulesEscapingDashes() {
+  const exportMap = {}
+  return rollup({
+    plugins: [
+      postcss({
+        include: '**/*.css',
+        sourceMap: true,
+        escapeDashes: true,
+        plugins: [
+          require('postcss-modules')({
+            getJSON(id, exportTokens) {
+              exportMap[id] = exportTokens
+            }
+          })
+        ],
+        getExport(id) {
+          return exportMap[id]
+        }
+      }),
+      babel({
+        babelrc: false,
+        presets: [['es2015', { modules: false }]],
+        include: '**/*.js',
+        sourceMap: true
+      })
+    ],
+    entry: path.resolve('./fixtures/fixture_modules.js')
+  }).then(bundle => {
+    const result = bundle.generate({
+      format: 'umd',
+      moduleName: 'default',
+      sourceMap: true
+    })
+    bundle.write({
+      dest: './output/output_modules.js',
+      moduleName: 'default',
+      format: 'umd',
+      sourceMap: true
+    })
+    return result.code
+  })
+}
+
+export function buildWithCssModulesEscapingDashesExportNamed() {
```

Comment:
Add escapeDashes, exportNamed as variables with default values so that we can test all the combinations.
In this way we can test issues like the one at: [index.js::153](https://github.com/egoist/rollup-plugin-postcss/pull/30/files#diff-1fdf421c05c1140f6d71444ea2b27638R153)